Functions.Templates/Documentation/sqlIn.md

```diff
@@ -0,0 +1,42 @@
+#### Settings for a Azure SQL input binding
```

Comment:
It'd be good to find out why this is set to header level 4 as well - is this nested in such a way that it shouldn't be header level 1-3?